hub/jupyterhub_config.py

```diff
@@ -36,8 +42,8 @@
 ]
 
 # The spawned containers need to be able to talk to the hub, ok through the proxy!
```

Comment:
I'm not sure what the comment means by `ok through the proxy`, but here we aren't using the proxy anymore since we're going directly to the `hub` service. I think this comment is outdated. Now, it should read something like:

    Here, we give spawned containers access to the Hub's API